diff --git a/libs/libevdev/Makefile b/libs/libevdev/Makefile
--- a/libs/libevdev/Makefile
+++ b/libs/libevdev/Makefile
@@ -6,7 +6,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libevdev
 PKG_VERSION:=1.13.6
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.freedesktop.org/software/libevdev/
@@ -26,7 +26,6 @@ define Package/libevdev
   CATEGORY:=Libraries
   TITLE:=a wrapper library for evdev devices
   URL:=https://www.freedesktop.org/wiki/Software/libevdev/
-  DEPENDS:=input-support
 endef
 
 define Package/libevdev/description
